BackofficeBundle/Resources/translations/interface.fr.yml

```diff
@@ -138,6 +138,12 @@ open_orchestra_backoffice:
                 L'édition d'une page en ligne est impossible. \n
                 Pour modifier  le contenu de la page veuillez créer une révision. \n
                 Vous pouvez également modifier le workflow de cette version mais la page ne sera plus visible sur le site
+        route_pattern_arguments:
+            title:  arguments manquants dans l'URL principale.
+            message: |
+                Cette page ne peut pas être publiée. \n
+                Certains blocs de cette page ont besoin de paramètres pour être afficher correctement. \n
+                Ces paramètres doivent être présents dans l'URL principale de la page. \n
```

Comment:
@itkg-nanne : you're talking about node in english and page in french which is not the same